tests/test_rtm.py

```diff
@@ -0,0 +1,345 @@
+from eventlet import sleep
+from eventlet.event import Event
+from mock import call, Mock, patch
+import pytest
+
+from nameko_slack import rtm
+
+
+@pytest.fixture
+def tracker():
+    yield Mock()
+
+
+@pytest.fixture
+def config():
+    return {}
+
+
+@pytest.fixture
+def service_runner(container_factory, config):
+    """
+    Service runner
+
+    Return a utility test function which runs the given service
+    and sets mocked Slack client to "publish" a given set of events
+
+    """
+
+    def _runner(service_class, events):
+
+        with patch('nameko_slack.rtm.SlackClient') as SlackClient:
+            SlackClient.return_value.rtm_read.return_value = events
+            container = container_factory(service_class, config)
+            container.start()
+            sleep(0.1)
```

Comment:
Why is the sleep required here?

Also, you could separate the concerns here into two fixtures -- one to apply the patch, and the other to start the service.